capdpa/IR/ir_unit.py

```diff
@@ -32,20 +41,5 @@ def PackagePath(self):
 class Unit(ir.Base):
 
     def UsedPackages(self):
-        types = []
-
-        def isLocalType(t):
-            return self.FullyQualifiedName()[:len(t.PackagePath())] != t.PackagePath() and \
-                   t.PackagePath()[:len(self.FullyQualifiedName())] != self.FullyQualifiedName()
-
-        for f in filter(ir_function.Function.isInst, self.children):
-            map(lambda p: types.append(p.ctype), f.parameters)
-            if f.return_type:
-                types.append(f.return_type)
-
-        map(lambda v: types.append(v.ctype), filter(ir_variable.Member.isInst, self.children))
-        map(lambda t: types.append(t.reference), filter(lambda c: ir_type.Type_Definition.isInst(c) and c.reference, self.children))
-        map(lambda c: types.append(c), filter(Class_Reference.isInst, self.children))
-        map(lambda t: types.append(t.ctype), filter(ir_variable.Variable.isInst, self.children))
-
-        return sorted(list(set(map(lambda t: t.PackageName(), filter(isLocalType, types)))))
+        packages = map(lambda x: x, self.UsedTypes(self.FullyQualifiedName()))
```

Comment:
Isn't `map(lambda x: x, <list>)` equivalent to just `<list>`?

Reply:
Good catch! There was an operation on `x` in an earlier version. Removed.